### ISSUE TITLE
Calculate sum of momentums in Tracker

### ISSUE THREAD

**Issue by michAtEl** [OpenSourcePhysics/tracker#14]
Hi,

I'd like to show the conservation of linear motion during a collision of two sliders on an air track.
I filmed the collision with an iphone with 240fps. I used tracker to measure the linear momentus p_1(t) and p_2(t) as functions of time. 
Is it possible to calculate the sum of these linear momentus in tracker: p(t) = p_1(t) + p_2(t)?
Is it possible to have a plot of p(t) = p_1(t) + p_2(t) in tracker?

I know that it is possible to define new variables for a point mass. But I did not find a way to use data from different point masses in these variables.

I'm not an english native speaker. If you don't understand what I mean, please tell me and I will try to explain better.

Thanks four your help
Michael

p.s.: I realy like Tracker. Thanks for this great software!

**Comment by dobrown** [comment]
Yes, you can get what you want by creating a Center of Mass track that includes the 2 sliders.  Then the momentum of the cm equals the sum of the momenta of the sliders. 

Other than this, you are right that there is no way in Tracker to define a new data variable that uses data from multiple tracks. However, you can always open the data in the Data Tool, copy and paste the momenta into a single tab, and define the variable there.

**Comment by michAtEl** [comment]
Ok, I will use center of mass at the moment.

Probably, a feature like this would make sense in tracker:
At the moment, all tables with data are connected to a point mass or a center of mass. It would be nice to have a user defined table that is not connected to a single mass or center of mass. In this user defined table, it could be possible to define variables which use data from other tables. The data of the user defined table could be shown in a diagram like data from a table that is connected to a point of mass.

With this feature, it would be possible to calculate the sum of momenta without the center of mass. This would be nice because my students learn about the conservation of momentum before they learn about the center of mass.
There might be other cases for this. It would for example be possible to calculate the sum of energy during a collision.

What do you think about this?
Are there other usecases for a feature like this?
Or is the effort for a feature like this to big?
